vulture/core.py

```diff
@@ -489,40 +489,35 @@ def generic_visit(self, node):
 
 
 def _parse_args():
-    def csv(option, _, value, parser):
-        setattr(parser.values, option.dest, value.split(','))
-    usage = """\
-usage: %prog [options] PATH [PATH ...]
-
-Paths may be Python files or directories. For each directory vulture
-analyzes all contained *.py files.
-"""
+    usage = "%(prog)s PATH [PATH ...] [options]"
     version = "vulture {0}".format(__version__)
-    parser = optparse.OptionParser(usage=usage, version=version)
-    parser.add_option(
-        '--exclude', action='callback', callback=csv, metavar='PATTERN',
-        type='string', default=[],
-        help=(
-            'Comma-separated list of paths to ignore (e.g.,'
-            ' *settings.py,docs/*.py). PATTERNs can contain globbing'
-            ' characters (*, ?, [, ]). Treat PATTERNs without globbing'
-            ' characters as *PATTERN*.'))
-    parser.add_option(
-        "--sort-by-size", action="store_true",
-        help="Sort unused functions and classes by their lines of code")
-    parser.add_option('-v', '--verbose', action='store_true')
-    parser.add_option(
-        '--min-confidence', action='store', type='int', default=0, help=(
+    parser = argparse.ArgumentParser(prog='vulture', usage=usage)
+    parser.add_argument(
+        'paths', action='store', nargs='*', metavar='PATH', help=(
```

Comment:
`nargs='+'`

Reply:
Yes, that should be the expected behavior, but I saw that originally vulture did not throw an error if run without any arguments - I just thought that it would be nice to continue it. Anyways, changing it now.

Reply:
Also, do we need to document it somewhere?

Reply:
This is documented by "PATH [PATH ...]" and I don't think we need to put it into the README.

Reply:
Ah, correct!